src/main/java/teammates/ui/controller/StudentProfilePageAction.java

```diff
@@ -26,8 +26,7 @@ protected ActionResult execute() {
         }
 
         StudentProfilePageData data = new StudentProfilePageData(account, sessionToken, isEditingPhoto);
-        statusToAdmin = "studentProfile Page Load <br> Profile: "
-                + SanitizationHelper.sanitizeForHtmlTag(account.studentProfile.toString());
+        statusToAdmin = "studentProfile Page Load <br> Profile: test";
```

Comment:
You cant just hardcode `test`.

In the #8717, `test` is mentioned because the google id in the screenshot is `test`.
So you need to display the `googleId`